tests/Unit/Models/CommentTest.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Tests\Unit;
+
```

Comment:
trong file này em test những gì?

Reply:
file này em test guarded property, quan hệ comment belongs to user, quan hệ comment morph many images, quan hệ đa cấp parent, children của comment ạ